.github/workflows/release.yaml

```diff
@@ -25,6 +26,18 @@ jobs:
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
       - name: Run chart-releaser
```

Comment:
Can we add `pre-release` and `release` to the names of these functions just so we can see easily which one of the two is triggered? 
Might be confusing to see two processes with the same name, as one will be skipped when the workflow is triggered

Reply:
Yes makes sense good suggestion, thanks.